### PR TITLE
Check for both empty values org_id and folder_id added

### PR DESCRIPTION
**Pull request by ingwarr**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -17,7 +17,14 @@
 resource "random_id" "random_hash_suffix" {
   byte_length = 4
 }
+resource "null_resource" "org_id_or_folder_id_is_empty" {
+  count = "${var.org_id == "" && var.folder_id != "" ? 1 : 0}"
 
+  provisioner "local-exec" {
```
Comment (kopachevsky) [inline main.tf]: what if folder_id == ''" and org_id is not? is it ok?

Comment (ingwarr) [reply main.tf]: ok, it was a typo, fixed

**Comment by aaron-lane** [comment]
Fixes #26.

File: main.tf [hunk 2]
```diff
@@ -17,7 +17,14 @@
 resource "random_id" "random_hash_suffix" {
   byte_length = 4
 }
+resource "null_resource" "org_id_or_folder_id_is_empty" {
```
Comment (aaron-lane) [inline main.tf]:
```suggestion
resource "null_resource" "org_id_and_folder_id_are_both_empty" {
```

**Comment by aaron-lane** [comment]
The CLA has been signed!

**Pull request merged**